proto/rill/admin/v1/api.proto

```diff
@@ -1639,6 +1644,13 @@ message UpdateUserPreferencesResponse {
   UserPreferences preferences = 1;
 }
 
+message DeleteUserRequest {
+  string email = 1 [(validate.rules).string.email = true];
+  string organization = 2 [(validate.rules).string.min_len = 1];
```

Comment:
This field doesn't make sense since the user may be a member of multiple orgs, and permanently deleting the user will/should remove all those relationships. It's also not populated by the CLI now.